mix.exs

```diff
@@ -55,7 +55,11 @@ defmodule Appsignal.Phoenix.MixProject do
       end
 
     [
-      {:appsignal_plug, ">= 2.0.8 and < 3.0.0"},
+      {
+        :appsignal_plug,
+        ">= 2.0.8 and < 3.0.0",
```

Comment:
```suggestion
        ">= 2.2.8 and < 3.0.0",
```

Probably need to change this. See https://github.com/appsignal/appsignal-elixir/pull/713#issuecomment-965345406

Reply:
Done! This is `appsignal-phoenix`, though, which depends on `appsignal-plug`, so I bumped `appsignal-plug`'s version number as well so we could target it here: it's `2.0.9`, not `2.2.8`.